website/docs/reference/commands/list.md

```diff
@@ -22,7 +22,7 @@ dbt ls
 See [resource selection syntax](node-selection/syntax) for more information on how to select resources in dbt
 
 **Arguments**:
-- `--resource-type`: This flag limits the "resource types" that dbt will return in the `dbt ls` command. By default, the following resources are included in the results of `dbt ls`: models, snapshots, seeds, tests, and sources.
+- `--resource-type`: This flag limits the "resource types" that dbt will return in the `dbt ls` command. By default, all resources are included in the results of `dbt ls` except for analysis.
```

Comment:
```suggestion
- `--resource-type`: This flag restricts the "resource types" returned by dbt in the 'dbt ls' command. By default, all resources are included in the results of `dbt ls," except for the analysis.
```

Reply:
ack! so for sure that was my bad, i prefer `dbt ls` (per style guide too) and i just forgot to use the code apostrophe 🙈 sorry about causing that confusion!

ok i updated your final sentence - what do you think?

Reply:
Looks great -- committed your suggestion 🙌